docs/docs/environment-variables.md

```diff
@@ -71,3 +71,4 @@ Reserved environment variables:
 
 * `NODE_ENV`
 * `PUBLIC_DIR`
+* `GATSBY_BUILD_DIR`
```

Comment:
Should this line be removed? As people _can_ override `GATSBY_BUILD_DIR` to set their preferred build directory.

Reply:
Yes you right, but in an other hand they could override node_env as well. In this case it mean that we use `GATSBY_BUILD_DIR` in gatsby and so it would create conflict to use it in an other way.